frontend/src/metabase/query_builder/components/filters/pickers/DatePicker/DatePicker.tsx

```diff
@@ -2,20 +2,28 @@
 import React from "react";
 import { t } from "ttag";
 import cx from "classnames";
-import moment from "moment-timezone";
 import _ from "underscore";
 
+import { isStartingFrom } from "metabase/lib/query_time";
```

Comment:
Wasn't able to remove usage of `isStartingFrom` because it is used in a `onFilterChange` down below. This ought to get removed eventually, but it'll require additional refactoring.

Reply:
Do we have a place to track these leftover tasks or WDYT about adding them to the task list in the original epic?